site/src/docs/components/highlight/code.mdx

```diff
@@ -0,0 +1,110 @@
+  <Highlight text="You may select an highlight from the article to be displayed here. Select an excerpt that you want the user to pay attention to." variant="large"/>
+  <Highlight text="Add an interesting quote here" type="quote" reference="First name Last name. Title."/>
+  <Highlight text="Add an interesting quote here" variant="small" type="quote" reference="First name Last name. Title."/>
+  <Highlight text="Add an interesting quote here" variant="large" type="quote" reference="First name Last name. Title."/>
+</>
+```
+
+```html
+  <div>
+    <div class="hds-highlight-wrapper">
+      <div class="hds-highlight" role="region">
+        <div class="hds-highlight--text">You may select an highlight from the article to be displayed here. Select an excerpt that you want the user to pay attention to.</div>
+      </div>
+    </div>
+    <div class="hds-highlight-wrapper--small">
+      <div class="hds-highlight" role="region">
+        <div class="hds-highlight--text hds-highlight--small">You may select an highlight from the article to be displayed here. Select an excerpt that you want the user to pay attention to.</div>
+      </div>
+    </div>
+    <div class="hds-highlight-wrapper--large">
+      <div class="hds-highlight" role="region">
+        <div class="hds-highlight--text hds-highlight--large">You may select an highlight from the article to be displayed here. Select an excerpt that you want the user to pay attention to.</div>
+      </div>
+    </div>
+    <div class="hds-highlight-wrapper">
+      <div class="hds-highlight" role="region">
+        <div class="hds-highlight--text hds-highlight--quote">Add an interesting quote here</div>
+        <div class="hds-highlight--reference">First name Last name. Title.</div>
+      </div>
+    </div>
+    <div class="hds-highlight-wrapper--large">
+      <div class="hds-highlight" role="region">
+        <div class="hds-highlight--text hds-highlight--large hds-highlight--quote">Add an interesting quote here</div>
+        <div class="hds-highlight--reference">First name Last name. Title.</div>
+      </div>
+    </div>
+    <div class="hds-highlight-wrapper--small">
+      <div class="hds-highlight" role="region">
+        <div class="hds-highlight--text hds-highlight--small hds-highlight--quote">Add an interesting quote here</div>
+        <div class="hds-highlight--reference">First name Last name. Title.</div>
+      </div>
+    </div>
+  </div>
+```
+
+</Playground>
+
+
+#### With custom accent line color
+
+<Playground>
+
+
+```jsx
+<>
+  <Highlight text="You may select an highlight from the article to be displayed here. Select an excerpt that you want the user to pay attention to." />
+  <Highlight text="Add an interesting quote here" reference="First name Last name. Title." type="quote"/>
+</>
+```
+
+</Playground>
+
+#### With custom text color
```

Comment:
I think this and custom accent line are still work in progress as they don't have the customisations 😄